### PR TITLE
Implement PRW autoconfiguration (stolen from SUSYTools)

### PR DESCRIPTION
**Pull request by kkrizka**
Implements automatic association of PRW config files with the PRW tools using code copied from SUSYTools. Set `m_autoconfigPRW` to use.

* The campaign is automatically determined from the tags in the dataset name
* The campaign can be overridden by setting `m_mcCampaign` to a comma separated list of campaigns (ie: if one wants to treat mc16a+mc16c as a super-dataset).

I am marking this WIP since I am still testing it.

### CODE REVIEW

File: Root/BasicEventSelection.cxx [hunk 1]
```diff
@@ -950,6 +982,115 @@ EL::StatusCode BasicEventSelection :: execute ()
   return EL::StatusCode::SUCCESS;
 }
 
+// "Borrowed" from SUSYTools
```
Comment (kratsg) [inline Root/BasicEventSelection.cxx]: Can you add a link to where you got this from (the GitLab location)?

Comment (kkrizka) [reply Root/BasicEventSelection.cxx]: done

**Comment by kkrizka** [comment]
@kratsg, what are the protections on master? I did `git push`, expecting my master (with some of the changes in this PR) to fail with some errors. However it did not and master is now updated..

**Comment by kratsg** [comment]
> @kratsg, what are the protections on master? I did git push, expecting my master (with some of the changes in this PR) to fail with some errors. However it did not and master is now updated..

screenshot 2018-01-31 08 15 58

Unclear, perhaps you're an administrator. I can fix this with a force push to remove the top commit.

**Comment by kratsg** [comment]
Fixed.

**Comment by kratsg** [comment]
@kkrizka is this still a WIP or ready to go?

**Comment by kkrizka** [comment]
I think it should be ready to go. Still checking some things, but at least the old behavior shouldn't be broken.

**Pull request merged**